trio/_core/_io_epoll.py

```diff
@@ -1,6 +1,7 @@
 import select
 import attr
 from collections import defaultdict
+from typing import DefaultDict
```

Comment:
I generally just use `Dict` for defaultdicts, since you rarely wind up accessing the `.factory` member

Reply:
Switched to `Dict`